bin/pytorch_inference/signal9.py

```diff
@@ -0,0 +1,181 @@
+import argparse
+import json
+import os
+import platform
+import random
+import stat
+import subprocess
+import time
+
+
+#
+# python3 signal9.py '/Users/davidkyle/Development/NLP Models/elser_2/elser_model_2.pt' --num_allocations=4
+#
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('model', help='A TorchScript model with .pt extension')
+    # The pipes are created by the C++ app
+    # The names must match those passed to app
+    parser.add_argument('--output_file', default='out.json')
+    parser.add_argument('--log_file', default='log.txt')
+    parser.add_argument('--num_threads_per_allocation', type=int, help='The number of inference threads used by LibTorch. Defaults to 1.')
+    parser.add_argument('--num_allocations', type=int, help='The number of allocations for parallel forwarding. Defaults to 1')
```

Comment:
We should probably include setting the cache size here. Does it default to 0 if `--cacheMemorylimitBytes` isn't passed?

Reply:
> Does it default to 0 if --cacheMemorylimitBytes isn't passed?

Yes, the default cache value is `0`:

```
  --cacheMemorylimitBytes arg           Optional memory in bytes that the 
                                        inference cache can use - default is 0 
                                        which disables caching
```